packages/react/src/TreeView/TreeView.features.stories.tsx

```diff
@@ -891,6 +898,57 @@ export const InitialFocus: Story = () => (
   </div>
 )
 
+export const FocusManagement: Story = () => (
```

Comment:
@siddharthkp added a story and test in [4df63b8](https://github.com/primer/react/pull/4484/commits/4df63b86924514b990351d088ab772647f36d19c)



https://github.com/primer/react/assets/41085564/fba66a13-56a5-4815-af9d-82bfa763a097

Reply:
Thank you! :heart: